src/content/en/2021/security.md

```diff
@@ -12,97 +12,101 @@ tomvangoethem_bio: Tom Van Goethem is a researcher at the <a hreflang="en" href=
+As part of this effort to improve security and reliability by adopting modern versions, TLS 1.0 and 1.1 have been <a hreflang="en" href="https://datatracker.ietf.org/doc/rfc8996/">deprecated by the Internet Engineering Task Force (IETF)</a> as of March 25, 2021. All upstream browsers have also either completely removed support or deprecated TLS 1.0 and 1.1. For example, Firefox has deprecated TLS 1.0 and 1.1 but has <a hreflang="en" href="https://www.ghacks.net/2020/03/21/mozilla-re-enables-tls-1-0-and-1-1-because-of-coronavirus-and-google/">not completely removed it</a> because during the pandemic, users might need to access government websites that often still run on TLS 1.0. The user may still decide to change `security.tls.version.min` in browser config to decide the lowest TLS version they want the browser to allow.
 
 {{ figure_markup(
   image="security-tls-version-by-site.png",
   caption="TLS versions usage for sites.",
-  description="Bar chart showing that on desktop 60.36% of sites use TLSv1.3, while 39.59% use TLSv1.2. On mobile the figures are 62.14% and 37.80% respectively. TLSv1.0, TLSv1.1 barely register though there is a very small amount of QUIC usage (0.05% on desktop and 0.06% on mobile).",
+  description="Bar chart showing that on desktop 60.4% of sites use TLSv1.3, while 39.6% use TLSv1.2. On mobile the figures are 62.1% and 37.8% respectively. TLSv1.0, TLSv1.1 barely register though there is a very small amount of QUIC usage (less than 0.1% on desktop and mobile).",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vR38ZfiZXxdGuzG4ywCEPKIU1Wl6E2bZwcQPyALavGq5q948gmWb8sT-Xo5T6K5z8smKPg6EKxV0JUI/pubchart?oid=1868638312&format=interactive",
   sheets_gid="1012588329",
   sql_file="tls_versions_pages.sql"
   )
 }}
 
-60.36% of pages in desktop and 62.14% of pages in mobile are now using TLSv1.3, making it the majority protocol version right now instead of TLSv1.2. The number of pages using TLSv1.3 has increased approximately 20% since [last year](../2020/security#protocol-versions) when we saw 43.23% and 45.37% respectively.
+60.4% of pages in desktop and 62.1% of pages in mobile are now using TLSv1.3, making it the majority protocol version over of TLSv1.2. The number of pages using TLSv1.3 has increased approximately 20% since [last year](../2020/security#protocol-versions) when we saw 43.2% and 45.4% respectively.
```

Comment:
```suggestion
60.4% of pages in desktop and 62.1% of pages in mobile are now using TLSv1.3, making it the majority protocol version over TLSv1.2. The number of pages using TLSv1.3 has increased approximately 20% since [last year](../2020/security#protocol-versions) when we saw 43.2% and 45.4% respectively.
```